### ISSUE TITLE
Quick find files in hidden directories

### ISSUE THREAD

**Issue by noamross** [rstudio/rstudio#4236]
When using the `Cmd/Ctrl + .` shortcut, I cannot find files in hidden directories.  For instance, in the following test directory structure, I can not get to `.test/anothertest.R` or `.test/test.yml`.

```
.
├── .gitignore
├── .test
│   ├── anothertest.R
│   └── test.yml
├── .test.R
└── tt.Rproj
```

I realize that some hidden directories such as `.git` and `.Rproj.user` contain a lot of files one would want to avoid in the search tree.  However, many projects have useful hidden directories, especially for configuration.  I use `.circleci/` and `.github/` regularly.  Perhaps a short list of hidden directory names could be blacklisted?  And/or search behavior could reflect the state of viewing hidden files in the Files pane?

**Comment by warnes** [comment]
The whitelist would need to be user-specified either as a configuration option, or as a configuration file (say `.rstudioignore `using the same syntax as `.gitignore`).

**Comment by pat-s** [comment]
unstale

**Comment by edavidaja** [comment]
!!

**Comment by cderv** [comment]
I agree @edavidaja this is a great feature request ! Especially that now we are using a lot github action and files inside `.github/` folder can't be accessed with Got To file shortcut.

**Comment by cderv** [comment]
I still find this useful to have, especially to quickly open / acess file in `.github/` folder
